pages/blog/swr-v2.en-US.mdx

```diff
@@ -0,0 +1,278 @@
+---
+image:
+description: 'We are excited to introduce SWR v2.0, including features to improve the use experience of your application.'
+date:
+---
+
+import Callout from 'nextra-theme-docs/callout'
+import Bleed from 'nextra-theme-docs/bleed'
+
+import Authors, { Author } from 'components/authors'
+import Video from 'components/video'
+
+# Announcing SWR 2.0
+
+<Authors date="">
+</Authors>
+
+We are excited to introduce SWR v2.0, including features to improve the use experience of your application.
+
+## What’s New
+
+### Better React 18 Support
+
+SWR now uses `useSyncExternalStore` and `startTransition` internally to improve React 18 support, which ensures stronger UI consistency in concurrent rendering with React 18. The change is an internal implementation detail, so developers using SWR do not have to care about the change.
+
+### isLoading State
+
+`isLoading` is a new return value that indicates if there's an ongoing request and no "loaded data". Previously, you have to check `data` and `error` are `undefined`, but now you can use the `isLoading` value.
+
+You can improve the user experience by combining with `isLoading` and `isValidating` like this.
```

Comment:
I thinnk we can only have a simple example here, and link to the page with details of how `isLoading` and `isValidating` work.

Reply:
@huozhi Thank you for your feedback! I've moved the details to the Understanding SWR page and simplified the isLoading example.